packages/fxa-auth-server/lib/routes/utils/otp.js

```diff
@@ -31,5 +32,22 @@ module.exports = (log, config, db) => {
         }
       );
     },
+
+    /**
+     * Helper function to simplify generating otp codes.
+     *
+     * @param secret
+     * @param otpOptions
+     * @returns number
+     */
+    generateOtpCode(secret, otpOptions) {
+      const authenticator = new otplib.authenticator.Authenticator();
+      authenticator.options = Object.assign(
+        otplib.authenticator.options,
```

Comment:
from meeting: Add empty object here not to override `otplib.authenticator.options`